### DISCUSSION TITLE
Help compiling sh1mmer

### DISCUSSION THREAD

**Discussion by pazza2234** [3kh0/ext-remover#440] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
I got the wax4windows and the shim and it said it sucseded but I cant find where is the output / where the file I nead to use the recovory tool on.  I think there shuld be a update to the sh1mmer tutorial.

**Comment by ehunt34** [comment]
You can't use Windows. Run ubuntu off of a USB or something.

**Reply by Aka-but-nice** [reply]
@pazza2234 You could download the same board as yours, this will give you a completely personal Chromebook.

**Comment by xHydra1337** [comment]
If and only if you really want to use windows, you have to manually inject BeautifulWorld into the shim, and even then, it still may not work... and if it doesn't, you have to delete the shim bin file, and re download it, because it quite literally destroys the shim. It took me a few hours to get it to work on windows just to see if I could... eventually I got it, but you're honestly better off running a VM on Ubuntu or other Linux based Os's, but really theres a million different methods people have for modding these shims, and ive seen it all work, some better than others.

**Reply by Aka-but-nice** [reply]
And also, it might not work with canceling the process, that type of file, may corrupt your usb

**Comment by pazza2234** [comment]
why cant someone just give a fully made thing i can flash to a usb stick, plug in and be done. and if not can you give me at least a tutorial. I am currently trying auto-sh1mmer on a linux vm, but if someone has a basic working method please let me know.

**Reply by Aka-but-nice** [reply]
> why cant someone just give a fully made thing i can flash to a usb stick, plug in and be done. and if not can you give me at least a tutorial. I am currently trying auto-sh1mmer on a linux vm, but if someone has a basic working method please let me know.

3kh0 posted one on his github, it shows sh1mmer.me and a youtube video with the tutorial

**Comment by pazza2234** [comment]
I got it to work. but now it is asking for auth key. I started a new discution named sh1mmer asking for auth key https://github.com/3kh0/ext-remover/discussions/443 so this one dose not go off track.
